client/client.go

```diff
@@ -58,7 +58,7 @@ import (
 )
 
 // DefaultVersion is the version of the current stable API
-const DefaultVersion string = "1.23"
+const DefaultVersion string = "1.25"
```

Comment:
This is unrelated to the actual change in this PR right?  Its just updating the version to what it is supposed to be.